sdk/template/template/package.json

```diff
@@ -23,7 +23,7 @@
     "extract-api": "tsc -p . && api-extractor run --local",
     "format": "prettier --write \"src/**/*.ts\" \"test/**/*.ts\" \"*.{js,json}\"",
     "integration-test:browser": "echo skipped",
-    "integration-test:node": "echo skipped",
+    "integration-test:node": "find dist-esm/test -name '*.spec.js' | xargs nyc mocha -r esm --require source-map-support/register --reporter ../../../common/tools/mocha-multi-reporter.js --timeout 180000 --full-trace",
```

Comment:
Why this?
`find dist-esm/test -name '*.spec.js' | xargs`

Reply:
Now that we have the tests in nested folders, we need to find these files somehow. This is the best I could come up with! I wonder if there are better ways.

Reply:
This is also not a Windows compatible command although it might work with git bash or if you put all the tools in your path (which I do, but is not required).

Reply:
I'm up for changing this to something universal! I have yet to find what though. I can invest some time doing that tomorrow morning. If anyone has an answer before that, it's appreciated! (but not required 😆)

Reply:
Some libraries have `node` and `browser` folders to hold platform specific tests so this would break them.

Reply:
I think this might be a job for @willmtemple's fancy `devtool`, just add a command in there for running tests that can be invoked from CLI, written in Node so that it's cross-platform.

Reply:
> I'm up for changing this to something universal! I have yet to find what though.

Go with the glob patterns that we use in our packages for now.

Reply:
The issue is that this is not the output of rollup, this is the output of tsc. I can't use the output of rollup for this. I also want to have tests divided in sub-folders (internal and public). I will use Will's dev tool for this!

Reply:
Oh, @willmtemple just showed me that Mocha supports glob patterns! So I changed this to just use the path with quotes `\"dist-esm/test/**/*.spec.js\"`.